client/js/lounge.js

```diff
@@ -50,7 +54,7 @@ $(function() {
 
 	$(".tse-scrollable").TrackpadScrollEmulator();
 
-	var favico = new Favico({
+	favico = new Favico({
```

Comment:
So, this is an issue here. You'll need to have the dependency at the top called Favico, and keep the var here, otherwise you are redefining the library to be a new variable, not great.